<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="3">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="spinBox_4"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Load Data</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QComboBox" name="comboBox">
     <property name="maximumSize">
      <size>
       <width>315</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Events for Training</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="spinBox_3">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Neural Net Mode</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Batch size</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Quantity of Epochs </string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Events for Test</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Neural Net</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="13" colspan="3">
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>350</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
